CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v1.23.0 (2024-09-17)
+
+## Orchestrator Changes
+
+### Kubernetes
+* Support Kubernetes NVIDIA Device Plugin configurations through API ([#4182])
+* Support NVIDIA Container Toolkit configurations through API ([#4182])
+
+## Build Changes
+* Update bottlerocket-sdk to 0.45 ([#4189])
+
+### Kit Changes
+* Update to bottlerocket-core-kit to v2.5.0 ([#4189])
```

Comment:
Nit: "Update to ... to" is awkward. Removing the first "to" would make it resemble "Update bottlerocket-sdk to...."